docs/simulation/calculations.yaml

```diff
@@ -36,9 +32,34 @@ POPs:
     population_growth: Sum of population_growth modifiers
     pop_growth: Sum of pop_growth modifiers
     global_population_growth/10: Sum of global_population_growth modifiers
-  Consciousness from literacy:
-    formula: defines.pops.CON_LITERACY * Plurality * POP literacy * (1 + literacy_con_impact)
-  Life needs scalar:
-    formula: (1 + Plurality) * (1 + 2 * Consciousness / defines.pops.PDEF_BASE_CON) * (1 + sum of goods_demand) * defines.pop.BASE_GOODS_DEMAND * POP size / 200000
-  Everyday & luxury needs scalar:
-    formula: Life needs scalar * (1 + number of inventions * defines.pops.INVENTION_IMPACT_ON_DEMAND)
\ No newline at end of file
+  Attitudes:
+    Consciousness from literacy: defines.pops.CON_LITERACY * Plurality * POP literacy * (1 + literacy_con_impact)
+  Needs:
+    Life needs scalar: (1 + Plurality) * (1 + 2 * Consciousness / defines.pops.PDEF_BASE_CON) * (1 + sum of goods_demand) * defines.pop.BASE_GOODS_DEMAND * POP size / 200000
+    Everyday & luxury needs scalar: Life needs scalar * (1 + number of inventions * defines.pops.INVENTION_IMPACT_ON_DEMAND)
+  POP income:
+    Pensions: 2 * pension_level * administrative efficiency * social spending slider * base life needs price * (1 - effective tax) * POP size / 200000
+    Unemployment subsidies: 2 * unemployment_benefit * administrative efficiency * social spending slider * base life needs price * (1 - effective tax) * POP size / 200000
+    Wages:
+      Minimum (worker) wage: 2 * minimum_wage * administrative efficiency * base life needs price * (1 - effective tax) * POP size / 200000
+      RGO wages:
+        Owners:
+          if minimum worker wages > normal worker wage:
+            formula: (RGO income - total worker income) * (1 - effective tax)
+          else:
+            formula: RGO income * min(0.5, 2 * owners / workers) * owner POP size / owners * (1 - effective tax) 
+            owners: total size of owner POPs in state
+            workers: number of employed workers in RGO
+            min(0.5: Hardcoded maximum of half the RGO income.
+            2*: Hardcoded value.
+        Workers:
+          if is_slave: 0
+          else:
+            if minimum wage > normal wage:
```

Comment:
Just to make sure I understand correctly, here (and above on line 47) `minimum wage` is the `Minimum (worker) wage` calculated on line `44`, and `normal wage` the amount calculated after the `else` on line 61? So in practice we'll calculate the `Minimum (worker) wage` and the `normal worker wage`, and if the minimum wage is bigger then we calculate and use the formulae on lines 48 and 59, otherwise we just use the already-calculated `normal worker wage` and calculate and use the formula on line 50?

Reply:
"minimum wage is the Minimum (worker) wage calculated on line 44"
Yes. Owners don't have minimum wage, that's why I added "(worker)".

"and normal wage the amount calculated after the else on line 61"
Yes. I struggled documenting this...

"So in practice we'll calculate the Minimum (worker) wage and the normal worker wage [...]"
Yes. Note that minimum wage differs **per country** and **pop type**, not per RGO, province, POP or anything else.